Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -19,31 +20,53 @@ int main()
     int *p = &x;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: " << "address of x in memory" << endl;
     cout << "Actual value of p: " << p << endl;
-    cout << "Predicted value &x: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value &x: "<< "address of x in memory" << endl;
     cout << "Actual value &x: " << &x << endl;
-    cout << "Predicted value of *p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of *p: " << "3" << endl;
     cout << "Actual value of *p: " << *p << endl;
-    
+
     foo(p, x);
-    
-    cout << "Predicted value of *p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+
+    cout << "Predicted value of *p: " << "42" << endl;
     cout << "Actual value of *p: " << *p << endl;
-    cout << "Predicted value of x: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of x: " <<"42"<< endl;
     cout << "Actual value of x: " << x << endl;
     foo(p, y);
 
-    cout << "Predicted value of *p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of *p: " << "42" << endl;
     cout << "Actual value of *p: " << *p << endl;
-    cout << "Predicted value of y: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of y: " << "4" << endl;
     cout << "Actual value of y: " << y << endl;
 
     //start writing mySwap here
-    
+    mySwap(x, y);
+
+
     return 0;
 }
 
+void mySwap(int a, int b)
+ {
+     cout <<"original value of x:" <<a <<endl;
+     cout <<"original value of y:"<<b <<endl;
```

Comment:
Make sure your spacing is consistent. so it would be:
`cout << "original value of x:" << a << endl;`
`cout << "original value of y:" << b << endl;`